### ISSUE TITLE
Wrong config for Gamma

### ISSUE THREAD

**Issue by IdoZilberberg** [orbs-network/orbs-network-go#1256]
**Describe the bug**

TL;DR
A Gamma node will be stuck on an old block because Node Sync is configured to timeout after 20 minutes.

Observed in test `TestGamma_LeanHelix`: The test fails in case 1 of the 4 nodes is late to start - this leads to 3 nodes closing block height=1 and progressing to height=2 (3 nodes are enough to reach consensus on a block). The remaining node cannot participate in consensus on H=2, nor can it close H=1 once all others progressed beyond it. The only way for it to catch up is Node Sync which will update it with block H=1, and let it take part in consensus of H=2. The Gamma config effectively prevented NodeSync from running.
The fix is to set the node sync timeout at 1 second in `TemplateForGamma()`

A secondary improvement is to decrease Leader Election timeout from 4000ms (production default) to 700ms, thus increasing liveness of the Gamma network under test. *This value must be lower than the NodeSync time anyway*. This is also in `TemplateForGamma()`.

**Steps To Reproduce**

**Expected behavior**

**Logs**

If applicable, provide the link to the log files that contain the issue.

**Screenshots**

**Environment:**

If applicable, add the information about that you are running and their versions

**Additional context**

If possible, provide the failing test the reveals the issue.
